service/spring.gradle

```diff
@@ -8,6 +8,9 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-jdbc'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
+	implementation 'org.springframework.security:spring-security-oauth2-core'
+	implementation 'org.springframework.security:spring-security-oauth2-client'
+	implementation 'org.springframework.boot:spring-boot-starter-cache'
```

Comment:
nit: alphabetize within the block, plz

Reply:
well… it is not alphabetized to start with (test comes before web) and does it really add any value?

Reply:
I am going to push back harder and say alphabetically ordering this is a waste of time. If `implementation` comes before `testImplementation` then that destroys the value of trying to find anything. People are just going to ctrl-f as these strings are impossible to read anyway.

I don't think there is value in having predictable order here.